### PR TITLE
Added AvatarActivityTest #85

### PR DESCRIPTION
**Pull request by opticod**
with some fixes and refactoring. #85

### CODE REVIEW

**Comment by alma-castant** [comment]
Please @opticod describe the fix in the PR body and commit message, not only a link to the issue

**Comment by opticod** [comment]
@alma-castant This is a Espresso based UI testing class of `AvatarActivity` .This will test different Views and Buttons shown up in different orientations including landscape and portrait ,and the working of different click events on Buttons.It will also test the retain of app state working properly or not during orientation change. 

I have done some refactoring with code also.Previously It was wrongly indented.
I have made one fix here:
`if (AvatarRoomActivity.avatarRoomInstance != null) {
                    AvatarRoomActivity.avatarRoomInstance.finish();
                }`
This will perform `finish()` only when `AvatarRoomActivity.avatarRoomInstance != null`  .

File: PowerUp/app/src/androidTest/java/powerup/systers/com/powerup/espresso/AvatarActivityTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+@LargeTest
+public class AvatarActivityTest {
+    @Rule
+    public ActivityTestRule<AvatarActivity> activityRule = new ActivityTestRule<>(
+            AvatarActivity.class);
+
+    @Test
+    public void viewTest() {
+        onView(withId(R.id.faceView)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
```
Comment (alma-castant) [inline PowerUp/app/src/androidTest/java/powerup/systers/com/powerup/espresso/AvatarActivityTest.java]: Why are you changing the orientation after you check if the items are displayed and not before?

Comment (opticod) [inline PowerUp/app/src/androidTest/java/powerup/systers/com/powerup/espresso/AvatarActivityTest.java]: In some cases , we can observe that on rotation of screen some views gets hidden or not shown properly due to improper layout setting.

**Comment by aanchal07** [comment]
Closing the pull request since there are conflicts.

**Pull request closed without merging**